dotcom-rendering/src/lib/contributions.ts

```diff
@@ -218,22 +218,16 @@ export const hasCmpConsentForBrowserId = (): Promise<boolean> =>
 		});
 	});
 
-const twentyMins = 20 * 60000;
-export const withinLocalNoBannerCachePeriod = (): boolean => {
-	const item = window.localStorage.getItem(NO_RR_BANNER_TIMESTAMP_KEY);
-	if (item && !Number.isNaN(parseInt(item, 10))) {
-		const withinCachePeriod = parseInt(item, 10) + twentyMins > Date.now();
-		if (!withinCachePeriod) {
-			// Expired
-			window.localStorage.removeItem(NO_RR_BANNER_TIMESTAMP_KEY);
-		}
-		return withinCachePeriod;
-	}
-	return false;
-};
+export const withinLocalNoBannerCachePeriod = (): boolean =>
+	!!storage.local.get(NO_RR_BANNER_TIMESTAMP_KEY);
```

Comment:
it looks like we're losing some logic here

Reply:
I meant to keep this in a separate PR

Actually, the behaviour is identical because it uses libs’s `storage` expiry function, which automatically remove keys if they are accessed after the period, so this whole method can be simplified.

Reply:
oh nice!

Reply:
See #10048